Added slight_TLC5957 library

s-light opened the pull request:
TODO: define naming convention for example files

in my library there are some examples and development helpers that are 
not starting with the library name -
in the community_bundle build all examples are thrown together in one example folder for all libraries - 
is this a good idea??
if every library has a `simple_example.py` that does not work...

please let me know what you think of this. 
for me iam fine with renaming my examples so they all start with the library name.

tannewt commented:
I believe we do have a standard for example names that includes the library name. It's done this way because its expected that the files will move. @sommersoft and @kattni will know exactly what the normal form is.

Thanks for the library!

s-light commented:
thanks for the info! 
i *thought* i have read all the tutorials and guides carefully but i can't remember that this was in there somewhere ;-)
i have renamed the examples.
here is the next question regarding the library name..
i prefixed the library with my `slight`name (i have done this also with all my arduino libraries in the past)
the idea for me was - if someone else wants to create a library - i think its fair to not use only the chip-name... 
→ i have renamed all examples to match the library name..

tannewt commented:
Thanks! We like that prefix style as well.
